cluster/charts/crossplane/README.md

```diff
@@ -88,166 +88,6 @@ helm install crossplane --namespace crossplane-system crossplane-master/crosspla
 helm install crossplane --namespace crossplane-system crossplane-master/crossplane --version <version> --devel --disable-openapi-validation
 ```
 
-## Installing Infrastructure Providers
```

Comment:
Why are these parts removed?

Reply:
@muvaf they will be changing with new package manager, I'll circle back to update them in a subsequent PR 👍